src/NzbDrone.Core/MediaFiles/EpisodeImport/ImportDecisionMaker.cs

```diff
@@ -32,6 +36,7 @@ public class ImportDecisionMaker : IMakeImportDecision
                                    IMediaFileService mediaFileService,
                                    IDiskProvider diskProvider,
                                    IVideoFileInfoReader videoFileInfoReader,
+                                   ISampleService sampleService,
```

Comment:
We already have a sample spec, which uses this "service" I extracted this because of where its used.